frontend/src/components/layouts/Navbar.tsx

```diff
@@ -170,7 +168,7 @@ export default function Navbar() {
         <Button
           color="inherit"
           variant="outlined"
-          onClick={() => router.push('/login')}
+          onClick={(e) => handleMenuLinkClick(e, '/login')}
```

Comment:
what is e here?

Reply:
the mouse event (in this case, the click). It's needed to delay the navigation so that we can close the menu first. There might be a simpler way to do it

Reply:
ehh, not worth simplifying right now. I tried using material ui's menu, since they had one example of a navbar-like menu. The sample code wasn't displaying properly when I copied it